app/src/main/java/org/greenstand/android/TreeTracker/database/TreeTrackerDAO.kt

```diff
@@ -33,9 +33,15 @@ interface TreeTrackerDAO {
     @Query("SELECT * FROM planter_info WHERE _id = :id")
     suspend fun getPlanterInfoById(id: Long): PlanterInfoEntity?
 
+    @Query("SELECT * FROM planter_info WHERE planter_identifier = :identity")
+    suspend fun getPlanterInfoById(identity: String): PlanterInfoEntity?
```

Comment:
Can we rename this to getPlanterInfoByIdentifier? Otherwise it can get confusing with the other method of the same name.

Reply:
Okay I didn't know there was an existing method with same name